lib/web/ui/server.go

```diff
@@ -82,15 +84,20 @@ func MakeServers(clusterName string, servers []types.Server, accessChecker servi
 			return nil, trace.Wrap(err)
 		}
 
-		uiServers = append(uiServers, Server{
+		s := Server{
 			ClusterName: clusterName,
 			Labels:      uiLabels,
 			Name:        server.GetName(),
 			Hostname:    server.GetHostname(),
 			Addr:        server.GetAddr(),
 			Tunnel:      server.GetUseTunnel(),
 			SSHLogins:   serverLogins,
-		})
+		}
+		if server.GetCloudMetadata() != nil {
+			s.AWS = server.GetCloudMetadata().AWS
+		}
```

Comment:
Are we guaranteed a single server with AWS data? Is it intended for the last one to win?

Could we have a comment here explaining it, please?

Reply:
Sorry, I'm not sure I understood the question
Are you asking if each server instance will match a single AWS EC2 instance?
If that's the question, then yes :+1:

Reply:
It makes no sense because I misread the loop. Looks good, don't mind me. :)